Content.Server/AME/Components/AMEControllerComponent.cs

```diff
@@ -183,6 +186,18 @@ private void OnUiReceiveMessage(ServerBoundUserInterfaceMessage obj)
 
                 if (msg.Button == UiButton.ToggleInjection)
                     _adminLogger.Add(LogType.Action, LogImpact.Extreme, $"{_entities.ToPrettyString(mindComponent.Owner):player} has set the AME to {humanReadableState}");
+
+                // Admin alert
+                if (mindComponent.HasMind)
+                {
+                    var adminSystem = _entities.System<AdminSystem>();
+                    var antag = mindComponent.Mind!.UserId != null
+                                && (adminSystem.GetCachedPlayerInfo(mindComponent.Mind!.UserId.Value)?.Antag ?? false);
+                    if (GetCoreCount() * 2 == InjectionAmount - 2
+                        && msg.Button == UiButton.IncreaseFuel)
+                        _chat.SendAdminAlert(
+                            $"{mindComponent.Mind!.Session?.Name}{(antag ? " (ANTAG)" : "")} increased AME over safe limit to {InjectionAmount}");
+                }
```

Comment:
Wouldn't it be better to add an overload of the `SendAdminAlert` method that accepts the players uid, the message and an optional mind component which resolves the mind component and prepends the name infront of the message?
```cs
SendAdminAlert(EntityUid player, string message, MindComponent? mindComponent = null)
{
    if(!Resolve(player, ref mindComponent))
    {
        SendAdminAlert(message);
        return;
    }

    var adminSystem = _entities.System<AdminSystem>();
    var antag = mindComponent.Mind!.UserId != null
                                && (adminSystem.GetCachedPlayerInfo(mindComponent.Mind!.UserId.Value)?.Antag ?? false);
    
    SendAdminAlert($"{mindComponent.Mind!.Session?.Name}{(antag ? " (ANTAG)" : "")} {message}");
}
```